internal/apis/certmanager/validation/issuer.go

```diff
@@ -244,6 +244,11 @@ func ValidateCAIssuerConfig(iss *certmanager.CAIssuer, fldPath *field.Path) fiel
 			el = append(el, field.Invalid(fldPath.Child("ocspServer").Index(i), ocspURL, "must be a valid URL, e.g., http://ocsp.int-x3.letsencrypt.org"))
 		}
 	}
+	for i, issuerURL := range iss.IssuingCertificateURLs {
+		if issuerURL == "" {
+			el = append(el, field.Invalid(fldPath.Child("caIssuers").Index(i), issuerURL "must be a valid URL, e.g., http://ca.letsencrypt.org/ca.crt"))
```

Comment:
**note:** There's a syntax here reported by tests:

> ../../internal/apis/certmanager/validation/issuer.go:249:81: syntax error: unexpected literal "must be a valid URL, e.g., http://ca.letsencrypt.org/ca.crt" in argument list; possibly missing comma or )

**suggestion:** `fldPath.Child("caIssuers")` still has the old name here.

**nitpick (non-blocking):** I think it would be OK to leave out the example here. I recognise that the OCSP error above includes an example, but I think we'd be totally fine to leave one out here. Not a blocker though!

(suggestion includes syntax error fix)

```suggestion
			el = append(el, field.Invalid(fldPath.Child("issuingCertificateURLs").Index(i), issuerURL, "must be a valid URL"))
```

Reply:
Fixed